EU4toV2/Data_Files/blankMod/output/common/countries.txt

```diff
@@ -346,8 +346,9 @@ DMS		="countries/Desmond.txt"
 ENL		="countries/England.txt"#
 KID		="countries/Kildare.txt"
 HSC		="countries/Highlands.txt"
-LAN		="countries/Lancaster.txt"##
 LEI		="countries/Leinster.txt"##
+LNC		="countries/Lancaster.txt"##
```

Comment:
You can remove both LNC and LOI lines as they're not strictly needed. If the output has Lancaster, it'll be correctly set already; if not, there is no need for it, as we summon not these tags in decisions or events.
If you want to keep them, however, you must add the relevant localisations on 0_Names.csv